Source/WebCore/Modules/mediastream/gstreamer/GStreamerRtpSenderBackend.h

```diff
@@ -65,25 +65,11 @@ class GStreamerRtpSenderBackend final : public RTCRtpSenderBackend {
         );
     }
 
-    void clearSource()
-    {
-        ASSERT(hasSource());
-        m_source = nullptr;
-    }
-
-    void setSource(Source&& source)
-    {
-        ASSERT(!hasSource());
-        m_source = WTFMove(source);
-        ASSERT(hasSource());
-    }
-
-    void takeSource(GStreamerRtpSenderBackend& backend)
-    {
-        ASSERT(backend.hasSource());
-        setSource(WTFMove(backend.m_source));
-    }
+    void clearSource();
+    void setSource(Source&&);
+    void takeSource(GStreamerRtpSenderBackend&);
 
+    void startSource();
```

Comment:
Why did you move this from private to public? It does not seem to be needed for this patch, right?